CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.5.1-dev0
+
+### Enhancements
+
+* Default `download_dir` now lives under `$HOME/.cache/unstructured/ingest` instead
+of in the root directory under the name of `tmp-ingest-...`.
+
+### Features
+
+### Fixes
+
+* As of the caching update, now files are not downloaded over and over when 
+`download_dir` is not specified since its path is not random anymore.
```

Comment:
Nits: again let's specified for `unstructured-ingest` but also "if invoked with --preserve-downloads". Also "are not downloaded over and over" -> "are not re-downloaded"